### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by jzsmoreno**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by jzsmoreno — COMMENTED** [review]
approve changes to my own pull request

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (jzsmoreno) [inline index.html:50]: single comment

**Review by MajFons — CHANGES_REQUESTED** [review]
Request changes

File: index.html (hunk 1 above)
Comment (jzsmoreno) [inline index.html:50]:
```suggestion
<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```

**Review by jzsmoreno — COMMENTED** [review]
feedback

**Review by MajFons — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (jzsmoreno) [inline index.html]:
```suggestion
<h2 hidden>Game over, refresh to play again to track the feedback 🧑‍💻 🤖!</h2>
```

**Pull request merged**